### PR TITLE
update project to .net core 3.0

### PR DESCRIPTION
**Pull request by lillo42**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ env:
     - secure: "LK5FHdPiguTxnhEpOHIabuO7CBYCwCH6liJad+6NooqbTYfhFfwArAVWRS6MsJ4JXMZEI6JTHH9z9be/U1c2TplHUBuWubNIHSioE2jp1fsdXFMMxk1BZSSe9uaT2hSrS+8QKnWHazVISBsbYakgsiKrhY7dlhkKdjBCqR+hQdNwnZVTs8hVUZxASye59aM8LMtonnYhqSjq3Wbt3hmBs+tqTQRpUH1m5uiWeXg1HE0Ccv+TMXPUdiLUEGquNx1FHaS2Kskl8ij5Ya50v/ujD8SgxbAFaf66nnk8InVDfA+XKsu5t4IayprY0yw0FvKJ9DzDNoNLbWFJv3Ak2J002CF+5L+GSZt/j1mPEfU7sBkxORplyi7NreFcSwcK8xSxsJkbDa6kNNAL3APHn1QzVaOh5JKOFWfkz5V/USWDiBCPzTA1Ly9O5JjCR9gla+t827Hj63lcDE3c05wD4NTchoyKi2Fdtf8XY4QcsAuonKCmFSBf2U0utG181g1JcVA1/lN+hM7tKrk0m/ZbTmZ8XkqH0+K/0k3ezhQNLZ13Q2lYmHXmU9+ONyFMAdKJ+BSCe/2bkFh5+c9sXrFp0GPzWvnP9FerMkbNqx68hdtUoflUaYfrG9XVI6mFH+H+YE17Jysc70U1FATRaKOLiu0KLrmsj2pxdghWg4fR0rbRjyU="
 
 before_install:
-- wget https://download.visualstudio.microsoft.com/download/pr/c624c5d6-0e9c-4dd9-9506-6b197ef44dc8/ad61b332f3abcc7dec3a49434e4766e1/dotnet-sdk-3.0.100-preview7-012821-linux-x64.tar.gz -O dotnet-sdk-latest-linux-x64.tar.gz
+- wget https://download.visualstudio.microsoft.com/download/pr/c624c5d6-0e9c-4dd9-9506-6b197ef44dc8/ad61b332f3abcc7dec3a49434e4766e1/dotnet-sdk-3.0.100-linux-x64.tar.gz -O dotnet-sdk-latest-linux-x64.tar.gz
```
Comment (tmds) [inline .travis.yml]: Travis failed:
> $ wget https://download.visualstudio.microsoft.com/download/pr/c624c5d6-0e9c-4dd9-9506-6b197ef44dc8/ad61b332f3abcc7dec3a49434e4766e1/dotnet-sdk-3.0.100-linux-x64.tar.gz -O dotnet-sdk-latest-linux-x64.tar.gz
--2019-09-29 13:24:49--  https://download.visualstudio.microsoft.com/download/pr/c624c5d6-0e9c-4dd9-9506-6b197ef44dc8/ad61b332f3abcc7dec3a49434e4766e1/dotnet-sdk-3.0.100-linux-x64.tar.gz
Resolving download.visualstudio.microsoft.com (download.visualstudio.microsoft.com)... 93.184.215.201, 2606:2800:11f:7de:d31:7db:168f:1225
Connecting to download.visualstudio.microsoft.com (download.visualstudio.microsoft.com)|93.184.215.201|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-09-29 13:24:49 ERROR 404: Not Found.

**Comment by tmds** [comment]
@lillo42 I'm curious. Are you using this in production? Do you do some benchmarks that compare it with the default Transport?

**Comment by lillo42** [comment]
@tmds unfortunately I’m not using yet, where I work, we use Windows, but I’m trying find some one is using Linux and Asp Net Core to see if it’s possible using this lib and comparer

**Comment by tmds** [comment]
Thanks @lillo42 !

**Pull request merged**